### PR TITLE
catalog grid

### PR DESCRIPTION
**Pull request by plankton9991**
- [DEMO LINK](https://plankton9991.github.io/layout_catalog_grid/)

Додав паддінги щоб було більше простору перед зміною гріда

### CODE REVIEW

**Review by RuslanaTomnyuk — APPROVED** [review]
You need to replace `flex` with `grid` as I understood from the instructions, but your card still has flex!

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
fix comment from @RuslanaTomnyuk

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, 200px);
 
-  max-width: 944px;
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  max-width: 1004px;
```
Comment (AlinaMatuschak) [inline src/styles/_catalog.scss:10]: Is it necessary?

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+  gap: 48px;
   margin: 0 auto;
+  padding: 0 30px;
```
Comment (AlinaMatuschak) [inline src/styles/_catalog.scss:13]: you don't need add paddings

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
```
Comment (AlinaMatuschak) [inline src/styles/_catalog.scss:19]: You set card width on grid-template-columns

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
- from 768px to 1024px should be three columns
Screenshot 2022-03-15 at 18 34 26

- from 488px to 768px should be two columns
Screenshot 2022-03-15 at 18 35 50
Screenshot 2022-03-15 at 18 36 41

**Review by AlinaMatuschak — APPROVED** [review]
Good work 💪

**Pull request closed without merging**